org.dash.dash-core.metainfo.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.dash.dash-core</id>
+  <name>Dash Core</name>
+  <project_license>MIT</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <summary>
+    Fully validating Dash peer-to-peer network node, wallet and GUI
+  </summary>
+  <url type="homepage">
+    https://dash.org/
+  </url>
```

Comment:
```suggestion
  <url type="homepage">https://dash.org/</url>
```

Reply:
Did you autoformat this or something? This must not have linebreaks like that unless it's inside description.

Reply:
Just pulled it from https://github.com/flathub/org.bitcoincore.bitcoin-qt/blob/master/org.bitcoincore.bitcoin-qt.metainfo.xml

Reply:
Yea fix the linebreaks first. That one is also wrong.

Reply:
two wrongs don't make a right.

Reply:
Yes, I will have some pushes coming, just answering how they got there